src/popup/router/pages/Names/AuctionBid.vue

```diff
@@ -0,0 +1,114 @@
+<template>
+  <div class="popup">
+    <h4>{{ $t('pages.names.auctions.bid-on') }} {{ name }}</h4>
+    <AmountSend
+      :amountError="!+amount"
+      @changeAmount="val => (amount = val)"
+      :value="amount"
+      :errorMsg="amountError"
+    />
+    <Button extend @click="bid" :disabled="!!amountError || !+amount">
+      {{ $t('pages.names.auctions.bid') }}
+    </Button>
+
+    <div class="details-row">
+      <span>
+        {{ $t('pages.names.auctions.current-heighest-bid') }}
+      </span>
+      <span> {{ highestBid }} {{ $t('pages.appVUE.aeid') }} </span>
+    </div>
+    <div class="details-row">
+      <span>
+        {{ $t('pages.names.auctions.remaining-time') }}
+      </span>
+      <span>
+        {{ blocksToRelativeTime(expiration - topBlockHeight) }}
```

Comment:
Can we use it as a filter, like:
```suggestion
        {{ expiration - topBlockHeight | blocksToRelativeTime }}
```
?